sui/src/benchmark/bench_types.rs

```diff
@@ -1,21 +1,21 @@
 // Copyright (c) 2022, Mysten Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
+use super::load_generator::calculate_throughput;
 use clap::*;
 use std::default::Default;
+use std::path::PathBuf;
 use strum_macros::EnumString;
 use sui_network::transport;
 
-use super::load_generator::calculate_throughput;
-
 #[derive(Debug, Clone, Parser)]
 #[clap(
     name = "Sui Benchmark",
     about = "Local test and benchmark of the Sui authorities"
 )]
 pub struct Benchmark {
-    /// Size of the Sui committee. Minimum size is 4 to tolerate one fault
-    #[clap(long, default_value = "10", global = true)]
+    /// Size of the Sui committee.
+    #[clap(long, default_value = "1", global = true)]
```

Comment:
Why are we setting this to default = 1? Then someone who runs the bench with default values will get no sense of the overhead of running even with the smallest meaningful committee of 4 (in terms of sig verification etc).

Reply:
In setup, we spin out {committee_size} validators. However currently the benchmarking only support one validator, hence it will continuously looking for peers and get connection errors. Soon we should be able to add support for multiple validators though. 

For the default value, I'm changing it to 1 so we don't have the issue above. I can keep it at 10 or 4 but we just need to specify `--committee-size=1` for now

Reply:
@gdanezis I'm keeping it @ 1 for now. Happy to update it any time!